gson/src/main/java/com/google/gson/GsonBuilder.java

```diff
@@ -286,6 +286,16 @@ public GsonBuilder disableInnerClassSerialization() {
     return this;
   }
 
+  /**
+   * Configures Gson to include local and/or anonymous classes for serialization
+   * @return a reference to this {@code GsonBuilder} object to fulfill the "Builder" pattern
+   * @since 2.1
+   */
+  public GsonBuilder enableLocalOrAnonClassSerialization(){
```

Comment:
```suggestion
  public GsonBuilder enableLocalOrAnonClassSerialization() {
```

I think there should be a space to match the style of the rest of the project.